src/server/symbols.map

```diff
@@ -1,4 +1,4 @@
-MIR_SERVER_INTERNAL_2.21 {
+MIR_SERVER_INTERNAL_2.22 {
```

Comment:
@Saviq what are we doing with release/2.21? Could this be part of it?

Reply:
I'll pull `main` to `release/2.21` again next week, so yes it can be.